templates/accuracy_notice_email.html

```diff
@@ -0,0 +1,31 @@
+<p>
+    [Why me?] You are receiving this notification because you are listed as an owner of the following ChromeStatus feature entry:
```

Comment:
Sorry, I should have made it more clear in the design docs that the bracketed text was meant just to organize my thoughts about what goes into this message.   Let's leave off the phrases `[Why me?`], `[Why now?]`, `[Background/motivation]`, and `[Call to action]`.

Reply:
I see now! removed those.